lib/group/group.routes.js

```diff
@@ -0,0 +1,47 @@
+const Joi = require('joi');
+
+
+module.exports = {
+  routes: (models) => {
+    // const controllers = require('./group.controllers')(models);
+    const groupModels = require('./group.models');
+
+    return [
+        {
```

Comment:
indentation on this object is a bit messed up. opening bracket is indented once more than needed, and the rest of the object twice more.